front/lib/email.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * This file contains functions related to sending emails, as well as the
+ * content of emails themselves.
+ */
+import sgMail from "@sendgrid/mail";
+
+import { Membership, User, Workspace, XP1User } from "@app/lib/models";
+import logger from "@app/logger/logger";
+
+import { ModelId } from "./databases";
+
+const { SENDGRID_API_KEY, XP1_CHROME_WEB_STORE_URL } = process.env;
+
+if (!SENDGRID_API_KEY) {
+  throw new Error("Missing SENDGRID_API_KEY env variable");
+}
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+export async function getAdminEmailsForWorkspace(id: ModelId) {
```

Comment:
Could we pass the workspace sid with `getAdminEmailsForWorkspace(workspaceId: string) {`